qa/zcash/smoke_tests.py

```diff
@@ -227,15 +228,15 @@ def z_mergetoaddress(results, case, zcash, from_addrs, to_addr):
 def check_z_mergetoaddress(results, case, zcash, from_addrs, to_addr, amount):
     txid = z_mergetoaddress(results, case, zcash, from_addrs, to_addr)
     if txid is None:
-        return decimal.Decimal('0')
+        return [decimal.Decimal('0')]
```

Comment:
In this case this change does not seem correct. Review: is check_z_mergetoaddress used?

Reply:
This  method is not used.